include/onnxruntime/core/framework/op_kernel_context.h

```diff
@@ -130,6 +130,12 @@ class OpKernelContext {
    */
   Status GetTempSpaceAllocator(AllocatorPtr* output) const ORT_MUST_USE_RESULT;
 
+  /**
+   Return the allocator associated with the CPU EP with memtype of OrtMemTypeDefault.
+   @remarks Use SafeInt when calculating the size of memory to allocate using AllocatorPtr->Alloc.
+   */
+  Status GetCPUAllocator(AllocatorPtr* output) const ORT_MUST_USE_RESULT;
```

Comment:
Should this be called GetTempSpaceCPUAllocator since we have GetTempSpaceAllocator for non-CPU devices?

Reply:
Done